src/storage/src/hummock/compactor/mod.rs

```diff
@@ -729,7 +729,7 @@ impl Compactor {
         filter_key_extractor: Arc<FilterKeyExtractorImpl>,
         task_progress: Option<Arc<TaskProgress>>,
     ) -> HummockResult<(Vec<SplitTableOutput>, CompactionStatistics)> {
-        let builder_factory = RemoteBuilderFactory::<F, BloomFilterBuilder> {
+        let builder_factory = RemoteBuilderFactory::<F, XorFilterBuilder> {
```

Comment:
Do we need to support both `BloomFilter` and `XorFilter` by configure ? @hzxa21

Reply:
I don't have a strong option but I think it is okay to do so if supporting two by config is trivial. Just in case we find any issue with xor filter and need to switch back.